modules/sso/config/launch/setup/73/added/launch/openshift-x509.sh

```diff
@@ -37,6 +37,10 @@ function autogenerate_keystores() {
     local PASSWORD=$(openssl rand -base64 32)
     local JKS_KEYSTORE_FILE="${KEYSTORE_TYPE}-keystore.jks"
     local PKCS12_KEYSTORE_FILE="${KEYSTORE_TYPE}-keystore.pk12"
+    local KEYTOOL="$(which keytool 2>/dev/null)"
+    if [ ! -x "$KEYTOOL" -a -x "$JAVA_HOME/bin/keytool" ]; then
+        KEYTOOL="$JAVA_HOME/bin/keytool"
```

Comment:
@hmlnarik 
Wondering, should this be using the `keytool` from OpenJ9 JDK or JRE? AFAICT there seems to be two locations per:
```
# rpm -q adoptopenjdk-8-openj9
adoptopenjdk-8-openj9-8u232_b09.openj9_0.17.0-1.x86_64
```

```
# rpm -ql adoptopenjdk-8-openj9 | grep keytool
/usr/lib/jvm/adoptopenjdk-8-openj9/bin/keytool
/usr/lib/jvm/adoptopenjdk-8-openj9/jre/bin/keytool
/usr/lib/jvm/adoptopenjdk-8-openj9/man/ja/man1/keytool.1
/usr/lib/jvm/adoptopenjdk-8-openj9/man/ja_JP.UTF-8/man1/keytool.1
/usr/lib/jvm/adoptopenjdk-8-openj9/man/man1/keytool.1
```

The reason, why I am asking being, if the JDK path of the `keytool` would be used, the path provided to the `SYSTEM_CACERTS` variable below would not work. See also comment below.